doc/system_requirements.md

```diff
@@ -190,36 +249,84 @@ Needs: dsn
 
 ### Dropping Database Objects
 
-#### Dropping Generic Database Objects
-`req~dropping-generic-database-objects~2`
+### Dropping Users
+`req~dropping-users~1`
+
+Users can drop database users using TDDB.
+
+Covers:
+
+* `feat~dropping-existing-database-objects~2`
```

Comment:
Please use the back-link variant uniformly throughout the document. Since HL requirements and features are in the same file, you can always use links.